konami/io/usbio2.md

```diff
@@ -0,0 +1,24 @@
+# USBIO2
+
+* aka (IIDX) "IO2" PCB
+* 2nd generation USB IO board to connect cabinet inputs and outputs to the main PCB
+* Part number
+  * PCB: PWB1111362080000
```

Comment:
Missing `Interface to main PCB: USB 2.0` that was in the original overview here.